docs/00-manuscripts/ieee/reviews/reviewer2.md

```diff
@@ -35,32 +35,33 @@ Here Shannon Entropy is defined as `H(X)` which itself is defined as
 $H(X) = - \sum_{i=1}^{n} P(x_i)\log_2P(x_i)$
 
 as in [Shannon 1948](https://ieeexplore.ieee.org/document/6773024).
-The Shannon formula is more often recognized as the "information gain" formula.
+The Shannon formula is more often referred to as "information gain".
 
 We've added this additional information and the Shannon reference to the manuscript.
 
 > 4/There is no theoretical analysis in the experiments section. All the conclusions are obtained from observation. It is not convincing.
 
-We are sorry to hear that you are not satisfied with parts of the manuscript.
-Obtaining conclusions from results of a study is good scientific practice which is why we framed the discussion around it.
-The discussion section (5) discusses various aspect of this study, from data quality to performance results, the effect of feature selection methods over to how the resulting feature importance measures can be linked back to the spectral characteristics of the data.
+Thank you for these useful suggestions for improving this part of the manuscript. Given the unique characteristics of different applications, like detecting forest disease from hyperspectral imagery, it is necessary to learn from representative case studies, since there is not theoretical proof of the superiority of one algorithm in this application. Nevertheless, we believe that we have found clear empirical evidence of the superiority of one machine-learning model in this use case, and we have identified potentials but also limitations of feature selection techniques in this setting. We have attempted to highlight these findings more clearly in the revised manuscript.
```

Comment:
```suggestion
Thank you for these useful suggestions for improving this part of the manuscript.
Given the unique characteristics of different applications, like detecting forest disease from hyperspectral imagery, it is necessary to learn from representative case studies, since there is not theoretical proof of the superiority of one algorithm in this application.
Nevertheless, we believe that we have found clear empirical evidence of the superiority of one machine-learning model in this use case, and we have identified potentials but also limitations of feature selection techniques in this setting.
We have attempted to highlight these findings more clearly in the revised manuscript.
```

Thanks!